dist/stat_data.py

```diff
@@ -658,6 +658,7 @@ def __init__(self, name, desc, flags=''):
     TxnStat('txn_read_queue_len', 'read timestamp queue length'),
     TxnStat('txn_read_queue_walked', 'read timestamp queue entries walked'),
     TxnStat('txn_rollback', 'transactions rolled back'),
+    TxnStat('txn_rollback_force_evicting', 'transaction rolled back while force evicting due to being oldest'),
```

Comment:
I prefer this to be grouped with the force eviction statistics `cache_eviction_force_rollback` or similar.